duo_client/admin.py

```diff
@@ -2092,17 +2157,31 @@ def update_integration(self,
         response = self.json_api_call('POST', path, params)
         return response
 
-    def get_admins(self):
+    def get_admins(self, limit=None, offset=0):
         """
-        Returns list of administrators.
+        Retrieves a list of administrators.
+        Args:
+            limit: The max number of admins to fetch at once. Default None
+            offset: If a limit is passed, the offset to start retrieval.
+                    Default 0
 
-
-        Returns list of administrator objects.  See the adminapi docs.
+        Returns: list of administrators. See the adminapi docs.
 
         Raises RuntimeError on error.
+
         """
-        response = self.json_api_call('GET', '/admin/v1/admins', {})
-        return response
+
+        (limit, offset) = self.normalize_paging_args(limit, offset)
+        if limit:
+            return self.json_api_call(
+                'GET',
+                '/admin/v1/admins',
+                {'limit': limit, 'offset': offset}
+            )
+
+        generator = self.json_paging_api_call('GET', '/admin/v1/admins', {})
```

Comment:
is there a reason this one doesn't follow the pattern of having `get_admins_iterator()`

Reply:
actually it looks like #66 also adds get_admins. We should probably pick one of these to have it and rip it out of the others

Reply:
Addressed in main comment thread.